org-superstar.el

```diff
@@ -637,10 +637,10 @@ This function may be expensive for files with very large plain
 lists; consider using ‘org-superstar-toggle-lightweight-lists’ in
 such cases to avoid slowdown."
   (or org-superstar-lightweight-lists
-      (and (save-match-data
+      (save-match-data
+        (and (not (org-in-src-block-p))
 	     (org-element-lineage (org-element-at-point)
-				  '(plain-list) t))
-	   t)))
+				  '(plain-list) t)))))
```

Comment:
I think it would be nice to keep the `t` at the end, see the comment above the function.

Reply:
AFAIK that's not idiomatic Lisp.  Any value can be treated as nil or non-nil, so it's not necessary to explicitly return t unless the calling function expects to have a different meaning than any other non-nil value.

Reply:
In my experience predicates often merely guarantee to return _some_ non-nil value, absolutely.  Often their docstrings state "Return non-nil if..." or some variation for that reason.  Although I have seen plenty of predicates explicitly state to return t specifically ("Return t if.." being actually used as an example for a predicate docstring in the Elisp reference manual), in which case I'd argue the function should ensure that to be the case.   In my mind it's a case where coding style is influenced by documentation style.

Reply:
Some of those "return t if" predicates in Elisp are core functions implemented as C subrs, and there would be no reason for them to return anything other than t.

In a function like this, there's no special meaning to t as opposed to another non-nil value, and AFAIK there's no advantage to returning t explicitly, so the code would be simpler without explicitly returning t.

Not that it really matters here, but simpler is usually better.  :)